### PR TITLE
handled colors

### PR DESCRIPTION
**Pull request by chenKotler30**

### CODE REVIEW

File: Board.h [hunk 1]
```diff
@@ -32,11 +31,15 @@ class Board {
...
 
+	bool colored=true;
+
 	Board(int _player) : player(_player), gameZone(_player), gameFrame(gameZone), topB(new TopBoard(this)) { setBoard(); }; // ctr 
 
 	~Board() { delete topB; }; // dctr
 
-	
+	void setColored(bool _colored) {
+		colored = _colored;
```
Comment (Yinon178) [inline Board.h:42]: private variables usually use  abstraction functions like this one. colored is Public therefore you can access it directly. Maybe change it to private?

File: Point.cpp [hunk 2]
```diff
@@ -2,9 +2,12 @@
 
 using namespace std;
 
-void Point::draw(char ch) const
+void Point::draw(char ch,bool colored) const
 {
-	int colorShape = getSerialNumber() % 6 + 1;
+	int colorShape = 7;
+	if (colored) {
+		colorShape = getSerialNumber() % 7 + 1;
```
Comment (Yinon178) [inline Point.cpp:9]: Changing it to 7 doesn't change the options to contain white ? do we want that ?

File: TetrisGame.cpp [hunk 3]
```diff
@@ -38,8 +38,10 @@ void TetrisGame::run() {
 		if (_kbhit()) // checks if there is anything in the buffer
 		{
 			keyPressed = _getch();
-			if (keyPressed == START)
+			if (keyPressed == START || keyPressed == STARTNOCOLOR)
 			{
+				setColored(keyPressed == START);
```
Comment (Yinon178) [inline TetrisGame.cpp:43]: can you explain ?

File: TetrisGame.cpp [hunk 4]
```diff
@@ -16,7 +16,7 @@ enum Type{Single, SQ, R, P, L, Z, MZ, ML};
 
 enum Speed {Fast = 130, Normal = 200 };
 
-enum MenuKeys {START = '1', PAUSE = '\x1b', RESUME = '2',  FAST_SPEED = '3', NORMAL_SPEED = '4', EXIT = '9'};
+enum MenuKeys {START = '1', STARTNOCOLOR = '5', PAUSE = '\x1b', RESUME = '2',  FAST_SPEED = '3', NORMAL_SPEED = '4', EXIT = '9'};
```
Comment (Yinon178) [inline TetrisGame.cpp:19]: should be 8 I think.

File: Point.cpp (hunk 2 above)
Comment (chenKotler30) [reply Point.cpp:9]: yes, why not?

File: Board.h (hunk 1 above)
Comment (chenKotler30) [reply Board.h:42]: changed

**Pull request merged**